### ISSUE TITLE
typescript definition files

### ISSUE THREAD

**Issue by dazinator** [kendo-labs/knockout-kendo#242]
Hello,

Just wondering if anyone happens to know if there are typescript definition files available for knockout-kendo?

**Comment by IanYates** [comment]
Not really.  I use the typescript definitions from Kendo itself just fine.

What are you trying to achieve?

The only bit of typescript I've got that even wants to touch Knockout Kendo is to set some global options for the date picker.  Specifically that is

```ts
ko.bindingHandlers.kendoDatePicker.options = {
    format: 'dd/MM/yyyy',
    parseFormats: ['dd MM yyyy', 'dd.MM.yyyy', 'dd-MM-yyyy', 'dd/MM/yy', 'dd.MM.yy', 'dd-MM-yy', 'dd/MM/', 'dd.MM.', 'dd-MM-', 'dd/MM', 'dd.MM', 'dd-MM', 'd/MM', 'dd.MM', 'dd-MM'],
    
};

interface KnockoutBindingHandlers {
    kendoDatePicker: KnockoutBindingHandler;
}
```

Even then I didn't bother specifying the type of "options" as it's typed as any and I wasn't using it anywhere apart from this one spot.
